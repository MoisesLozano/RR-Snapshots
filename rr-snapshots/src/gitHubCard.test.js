import App from './App';
import renderer from 'react-test-renderer'
import GitHubCard from './components/GitHubCard';

test('renders a snapshot of Github card', () => {
  const tree = renderer.create(<GitHubCard/>).toJSON()
  expect(tree).toMatchSnapshot()
})