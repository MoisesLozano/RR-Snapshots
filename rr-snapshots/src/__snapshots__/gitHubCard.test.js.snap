// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of Github card 1`] = `null`;
